recipes/assist-adam-fork/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "assist-adam-fork" %}
+{% set version = "1.1.9a2" %}
```

Comment:
This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
This package is a temporary fork of another package. I've changed the pypi release to use post syntax and mirrored that here. As far as I can tell, that should be parseable in conda and shouldn't create any issues or require extra labels. Please let me know if this meets requirements. Thank you for all your help!

Reply:
Yes, understood. This is not a pre-release and I've created a version "1.1.9.post1" to indicate that. I believe that is a valid version string and there shouldn't be any issues there, correct? I did read the documentation